packages/wrangler/src/__tests__/helpers/string-dynamic-values-matcher.ts

```diff
@@ -0,0 +1,28 @@
+export type PatternReplacementPair = [string | RegExp, string];
+
+/**
+ * Sometimes, we might need to test strings that contain dynamic |
+ * random | generated data (such as file hashes, IDs, etc.).
+ *
+ * This helper function replaces the dynamic parts of such strings with
+ * static values, thus enabling us to properly test the contents of the
+ * string.
+ *
+ * see https://jestjs.io/docs/snapshot-testing#property-matchers
+ */
+export function replaceRandomWithConstantData(
```

Comment:
I'm open to better naming here, if applicable. same goes for the file name

Reply:
Fine by me! People can rename if they want to abstract it further when they re-use :)